src/commands/txCommands/pullAll.js

```diff
@@ -0,0 +1,90 @@
+const chalk = require('chalk');
+const child_process = require('child_process');
+const Rx = require('rxjs');
+const txlib = require('./util');
+const pullResourceTrns = require('./util/pullResourceTrns');
+
+const getProjectResourcesList$ =
+	txlib.resources$
+		.flatMap(Rx.Observable.from);
+
+const child_process$ = Rx.Observable.bindNodeCallback(child_process.exec);
+
+/**
+ * Creates git commit of what has been staged
+ *
+ * note: adjusting resource name in git commit message to prevent jira ticket
+ * from moving through workflow. We have a script which looks at all git commits
+ * to see if a ticket number is contained within it and if so associates a branch/PR
+ * with it. We don't want that to happen with tx pulls.
+ * @param  {String} resource resource slug to be used when we pull translations
+ * @param  {Observable} observer Observable of resource pull
+ * @return {Observable} Observable of shell process
+ */
+const commitResource = (resource, observer) =>
+	child_process$(`git commit -m "tx:pull for ${resource.replace(/-/g, '_')}" --no-verify`)
+		.flatMap(() => {
+			console.log(chalk.green(`- commited changes for '${resource}'`));
+			// start next tx:pull
+			return observer.complete();
+		});
+
+/**
+ * Called when a tx pull is complete for a particular resource
+ * @param  {String} resource resource slug to be used when we pull translations
+ * @param  {Observable} observer Observable of resource pull
```

Comment:
'Observer' means something very different than 'Observable' in Rx - best not to conflate the two